agent/proxycfg/connect_proxy.go

```diff
@@ -70,6 +70,18 @@ func (s *handlerConnectProxy) initialize(ctx context.Context) (ConfigSnapshot, e
 		return snap, err
 	}
 
+	// Get information about the entire service mesh.
+	err = s.cache.Notify(ctx, cachetype.ConfigEntryName, &structs.ConfigEntryQuery{
```

Comment:
Previously this was only needed for tproxy, now it's for everything except mesh-gateways.